hack/metric-linter/metric_name_linter.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+PROJECT_ROOT="$(readlink -e "$(dirname "${BASH_SOURCE[0]}")"/../)"
+cd "${PROJECT_ROOT}/metric-linter"
+
+# Run the git command to get the remote origin URL and extract the operator and sub-operator values
+url=$(git config --get remote.origin.url | awk -F.com/ '{print $2}' | awk -F. '{print $1}')
+operator=$(echo "$url" | awk -F/ '{print $1}')
+subOperator=$(echo "$url" | awk -F/ '{print $2}')
```

Comment:
@avlitman @machadovilaca I'm not sure if this is good practice to set the operator and sub operator name. It might be ok, but only for our specific CNV use case. What will happen if we add a new operator? Are we sure we would still use the same logic for the sub operator name?
I think we should pass it as a parameter. What do you think?

Reply:
I think we should run the command with something like:
metric_name_linter.sh --operator='kubevirt' --suboperator='hco'
Is that possible?

Reply:
Done